### PR TITLE
ex4_js-objects-part2

### PR DESCRIPTION
**Pull request by AleksProkofiev**
A function that looks for a property only in the prototype of the passed object is written; 
A function that creates an empty object without a prototype is written; 
Written a function that removes the first and last space in a line; 
A function is written that is a string with a capital first character; 
Written a function that looks for one line in another; 
Written a function that returns a string, with the capital first character of each word;  
A function is written that shortens the string and adds ellipsis  to the end, so that the length of the cut string together with ellipsis is equal to the given number; 
A function is written that returns this string but in lowerCamelCase notation; 
A function that takes a string to another string after the given word number is written; 
A function is written that returns an inverted string; 
A function is written that will count how many times each character occurs in a string; 
Written function that adds two numbers and the result is rounded to the 3rd decimal place; 
A function is written that returns a random number in the range 0 - 100; A function is written that returns an integer random number in the range of  min, max.

### CODE REVIEW

File: src/ex4_js-objects-part2/task-01.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+function hasPropertyInPrototype(propName, object) {
+  //var count = 0;
+  for (var key in object) {
+    if (!object.hasOwnProperty(propName) && object[propName]) {
+        return object[propName];
+    }
+  }return undefined;
```
Comment (aneelia) [inline src/ex4_js-objects-part2/task-01.js:7]: return undefined = return = отстутствие return вообще
лучше возвращать  false

File: src/ex4_js-objects-part2/task-03.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+function removeFirstAndLastSpaces(string) {
+    var newString = string.trim();
```
Comment (aneelia) [inline src/ex4_js-objects-part2/task-03.js]: не прикольно((

Comment (aneelia) [reply src/ex4_js-objects-part2/task-03.js]: сделай это задание по другому, пожалуйста

File: src/ex4_js-objects-part2/task-01.js (hunk 1 above)
Comment (AleksProkofiev) [reply src/ex4_js-objects-part2/task-01.js:7]: Эт для тестов

Comment (aneelia) [reply src/ex4_js-objects-part2/task-01.js:7]: ок

**Pull request merged**